### DISCUSSION TITLE
🔗 🚀 Feature Request: Make the Taxonomy Filter be context aware based on Custom Queries (Duplicate of #23418)

### DISCUSSION THREAD

**Discussion by steveindzine** [elementor/elementor#23637] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

If you use a custom query in a post loop and add a taxonomy filter to it, the filter will show terms for posts that aren't in the post loop, it would be a much better filter if it used the same posts.

### Describe the solution you'd like

The filter needs to use the same custom query and only show terms which are relevant to the query.

### Describe alternatives you've considered

facetwp is the best solution but is a premium plugin.

**Comment by nicholaszein** [comment]
Hello there, @steveindzine!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23418

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
